src/System.Private.CoreLib/shared/System/LocalAppContextSwitches.Common.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace System
+{
+    internal static partial class LocalAppContextSwitches
```

Comment:
This is refactored to match the pattern used by CoreFX. I am going to flip CoreFX to use this shared copy once it is mirrored.

Reply:
I assume you mean CoreRT.
Curious: Why is it called Local now?

Reply:
Both CoreRT and CoreFX actually.

It is called Local because of it is local cache of the switch values. I will add comment about it.